flocx_market/api/bid.py

```diff
@@ -1,47 +1,35 @@
 from flask_restful import Resource
 from flask import request
-from flocx_market.db.sqlalchemy import api as dbapi
+from flocx_market.objects.bid import Bid as bo
```

Comment:
I'd remove `as bo`, and just  do things such as `Bid.get(marketplace_bid_id)` below.

Reply:
The it causes some namespace issue, as my api class is also Bid, so when it tries to call get in Bid it goes into recursion. Do you think I should change the Bid class name in Objects.

Reply:
Ohhh - I'd just import flocx_market.objects.bid then, and call `bid.Bid.get()`